Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp

```diff
@@ -1893,6 +1895,66 @@ int CLuaGUIDefs::GUISetEnabled(lua_State* luaVM)
     return 1;
 }
 
+int CLuaGUIDefs::GUIBlur(lua_State* luaVM)
+{
+    //  bool guiBlur ( element guiElement )
+    CClientGUIElement* guiElement;
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(guiElement);
+
+    if (!argStream.HasErrors())
+    {
+        CStaticFunctionDefinitions::GUIBlur(*guiElement);
+        lua_pushboolean(luaVM, true);
```

Comment:
You can reduce the two lines into `lua_pushboolean(luaVM, CStaticFunctionDefinitions::GUIBlur(*guiElement));`